### PR TITLE
fill some void

### PR DESCRIPTION
**Pull request by maxtori**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,3 +1,4 @@
 _build/
 *.install
 *.merlin
+test
```
Comment (redianthus) [inline .gitignore:4]: I think you forgot to remove this line ?

File: dune-project [hunk 2]
```diff
@@ -18,11 +18,6 @@
...
-   (>= 4.05))
-  (dune
-   (>= 2.0))
-  (js_of_ocaml
-   (>= 3.6))
-  (js_of_ocaml-ppx
-   (>= 3.6))))
+  (ocaml (>= 4.05))
+  (dune (>= 2.0))
+  ezjs_min))
```
Comment (redianthus) [inline dune-project:23]: That looks wrong to me. E.g. in the dune file you got: `preprocess (pps js_of_ocaml-ppx))`.

File: src/dune [hunk 3]
```diff
@@ -1,5 +1,4 @@
 (library
  (public_name ezjs_push)
- (preprocess
-  (pps js_of_ocaml-ppx))
- (libraries js_of_ocaml))
+ (preprocess (pps js_of_ocaml-ppx))
+ (libraries ezjs_min bigstring base64))
```
Comment (redianthus) [inline src/dune:4]: Why are you removing js_of_ocaml ?

File: .gitignore (hunk 1 above)
Comment (maxtori) [reply .gitignore:4]: It doesn't hurt

File: dune-project (hunk 2 above)
Comment (maxtori) [reply dune-project:23]: ezjs_min will always imply js_of_ocaml-ppx/js_of_ocaml dependencies (as js_of_ocaml-ppx will always imply js_of_ocaml)

File: src/dune (hunk 3 above)
Comment (maxtori) [reply src/dune:4]: same

**Pull request merged**